src/shared/components/structureViewer/StructureViewerPanel.tsx

```diff
@@ -414,8 +422,11 @@ export default class StructureViewerPanel extends React.Component<IStructureView
                             pdbId={this.pdbId}
                             chainId={this.chainId}
                             residues={this.residues}
+                            bounds={this.structureViewerBounds}
+                            containerRef={(div: HTMLDivElement) => {
```

Comment:
its better practice to not use lambdas for rendering because that will cause unnecessary updates, since the anonymous function is recreated and so always different on each pass through. Better to hold onto this, maybe in a `handlers` object.